certificate-service/domain/src/main/java/se/inera/intyg/certificateservice/domain/action/model/CertificateActionFactory.java

```diff
@@ -143,6 +143,14 @@ public static CertificateAction create(CertificateActionSpecification actionSpec
               )
           )
           .build();
+      case SEND_AFTER_SIGN -> CertificateActionSendAfterSign.builder()
+          .certificateActionSpecification(actionSpecification)
+          .actionRules(
+              List.of(
+                  new ActionRuleStatus(List.of(Status.SIGNED))
```

Comment:
I'm wondering about the action rule status set to SIGN. For SIGN it is set to DRAFT. But maybe a certificate that is supposed to be signed and sent have both SIGN and SEND_AFTER_SIGN?

Reply:
If the certificate is a draft, only the sign-resourcelink will be provided.

If the certificate is signed, the SEND_AFTER_SIGN-resourcelink will be provided. This will need to be refactored when we introduce the ability to complement. But lets solve that issue when we introduce that functionality, and go with the simplest implemenation for now?